docs/application_logic/protocol.md

```diff
@@ -1,4 +1,46 @@
-# Protocol high level overview
+# Protocol high-level overview
 
-will link (in the future) to specific specs
-- Overall data flow might help here
+To orientate the reader, we provide a high-level overview of the protocol, following the natural life cycle of a message. The first module used -while the message is still being created-, is the **Tip Selection**(ADD LINK) module. Here, the node must choose a certain number (from two to some defined upper number) of other messages to reference, meaning that the newly created message will be cryptographically attached to these referenced messages. An honest node must always choose tips uniformly at random from a tip pool, i.e., from a set of still unreferenced messages that satisfy a certain set of conditions, as discussed on INSERT LINK. In the diagram below, the issuance process being described now is represented in the context of the complete protocol. 
```

Comment:
Is there any upper number already decided for selecting? I've seen in the tip selection draft there was 8

Reply:
yes, maybe it's better to just say 8 and change it later, if that's the case. :)